openapi.yaml

```diff
@@ -45,6 +45,11 @@ paths:
       responses:
         '200':
           description: Success.
+          headers:
+            FicAiSession:
```

Comment:
`FicAiSession` is not the name of a header, it's the name of a cookie

Reply:
Oops, thanks; fixed and added to create session